torctl

```diff
@@ -522,6 +559,65 @@ rvmac() {
     msg "reverted mac addresses"
 }
 
+chng_timezone() {
+    warn "changing timezone"
+    backup_timezone
+    timedatectl set-timezone UTC
+    msg "changed timezone"
+}
+
+rv_timezone() {
+    warn "reverting timezone"
+	tzb=$(cat /tmp/timezone.bak) && timedatectl set-timezone $tzb
+    msg "reverted timezone"
+}
+
+chng_hostname() {
+	warn "changing hostname"
+	array[0]="Windows10Enterprise "
+	array[1]="Windows10Pro"
+	array[2]="Windows10EnterpriseLTSC "
+	array[3]="Windows8.1OEM"
+	array[4]="Windows8Enterprise"
+	array[5]="Windows8.1Pro"
+	array[6]="Windows7Professional"
+	array[7]="Windows7Enterprise"
+	array[8]="Windows7Ultimate"
+	array[9]="WindowsVistaBusiness"
+	array[10]="WindowsXPProfessional"
+	array[11]="macOS11"
+	array[12]="OSX10.11"
+	array[13]="MacBook Air"
+	array[14]="MacBook"
+	array[15]="MacBook Pro"
+
+	size=${#array[@]}
+	index=$(($RANDOM % $size))
+
+	echo "127.0.0.1       localhost ${array[$index]}
+		::1             localhost ip6-localhost ip6-loopback
+		fe00::0         ip6-localnet
+		ff00::0         ip6-mcastprefix
+		ff02::1         ip6-allnodes
+		ff02::2         ip6-allrouters" > /etc/hosts 
```

Comment:
Do we realy need to change `/etc/hosts` in hostname changes? I personally doesn't see any valid reason to modify `/etc/hosts` in a hostame change. Please explain how it's needed.